aikau/src/main/resources/alfresco/layout/TitleDescriptionAndContent.js

```diff
@@ -95,10 +128,46 @@ define(["dojo/_base/declare",
        */
       postCreate: function alfresco_layout_TitleDescriptionAndContent__postCreate() {
          /*jshint eqnull:true*/
+         this.alfSubscribe(this.subscriptionTopic, lang.hitch(this, this.update), this.subscribeGlobal);
          if (this.widgets != null)
          {
             this.processWidgets(this.widgets, this.contentNode);
          }
+      },
+
+      /**
+       * 
+       * @instance
+       * @param {object} payload The payload containing the details to update
+       */
+      update: function alfresco_layout_TitleDescriptionAndContent__update(payload) {
+         if (this.itemKeyProperty && this.currentItem)
+         {
+            var itemKey = lang.getObject(this.itemKeyProperty, false, this.currentItem);
+            var payloadKey = lang.getObject(this.itemKeyProperty, false, payload);
+            if (itemKey && itemKey === payloadKey)
+            {
+               if (payload.title)
+               {
+                  this.titleNode.textContent = payload.title;
+               }
+               if (payload.description)
+               {
+                  this.descriptionNode.textContent = payload.description;
+               }
+            }
+         }
+         else
+         {
+            if (payload.title)
+            {
+               this.titleNode.textContent = payload.title;
+            }
+            if (payload.description)
+            {
+               this.descriptionNode.textContent = payload.description;
+            }
```

Comment:
Is this block meant to be identical to the one above? (probably is, but two identical multi-line blocks isn't very DRY, so just confirming!)